src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/example-global-iam.yml

```
@@ -34,6 +41,41 @@ Resources:
               - "*"
       Roles:
         - adf-cloudformation-deployment-role # This role is created in the global.yml and is the default cloudformation deployment role for ADF.
+  # ADFTerraformRole:
+  #   # This is the role that will be used to deploy Terraform resources from within 
+  #   # the target account. You should scope this policy depending on what you would 
+  #   # like to deploy within certain Organizational Units.
+  #   # NOTE: below is a sample IAM policy. This policies should NOT be used for purposes other than testing.
+  #   # Uncomment this line if you want to enable the terraform extensions
+  #   Type: AWS::IAM::Role
+  #   Properties:
+  #     RoleName: "adf-terraform-role"
+  #     AssumeRolePolicyDocument:
+  #       Version: "2012-10-17"
+  #       Statement:
+  #         - Effect: Allow
+  #           Sid: "AssumeRole"
+  #           Principal:
+  #             AWS:
+  #               - !Sub arn:aws:iam::${DeploymentAccountId}:role/adf-codebuild-role
```

This will need to change to:
```suggestion
  #               - !Sub arn:aws:iam::${DeploymentAccountId}:role/adf-terraform-role
```

---

If I understood properly you would create a adf-codebuild-terraform-role used by CodeBuild itself in the deployment account that assume adf-terraform-role in target account. Is that correct? Why wouldn't you use the adf-codebuild-role if the managed policy will be the same?